### ISSUE TITLE
redundant link in martini3001

### ISSUE THREAD

**Issue by fgrunewald** [marrink-lab/vermouth-martinize#525]
This link is never applied because it get's overwritten by another link. This is the intended behavior and just a remainder of development. 
https://github.com/marrink-lab/vermouth-martinize/blob/b450b6793e85e4f31542d9232b79a12c229a43ec/vermouth/data/force_fields/martini3001/aminoacids.ff#L1091-L1099

**Comment by pckroon** [comment]
I don't see which one overwrites it?

**Comment by fgrunewald** [comment]
yep really hard to see. Took me like 1h to figure it out. But this one:


https://github.com/marrink-lab/vermouth-martinize/blob/b450b6793e85e4f31542d9232b79a12c229a43ec/vermouth/data/force_fields/martini3001/aminoacids.ff#L1128-L1137

**Comment by fgrunewald** [comment]
yes but the bottom patterns include all above patterns the order in the ff file is incorrect

**Comment by pckroon** [comment]
I don't think it does?
`-BB BB {"cgsecstruct": "C", "resname": "PRO|HYP"} +BB` <!= 
`-BB BB {"cgsecstruct": "C", "resname": $protein_resnames_non_pro} +BB`
Note that it says `protein_resnames_non_pro`, so anything but PRO and HYP.

**Comment by fgrunewald** [comment]
Let's say we have the sequence `ALA-PRO-ALA`. What we expect is to have an angle `127 25` due to the first link. That matches because we have a Proline in the middle. But this match gets overwritten by the second pattern of the other link (i.e. `-BB {"cgsecstruct": "C", "resname": $protein_resnames_non_pro} BB +BB  `) This pattern only requires the first BB bead to be non-proline, which is true. Thus we get `127 20`. This is true for ALL cases. Because we never noticed, all itp files ever produced did not apply the first link, which therefore makes it redundant.

**Comment by pckroon** [comment]
Ooooooh you're right. Good catch :)

**Comment by fgrunewald** [comment]
@pckroon do we remove it or leave it?

**Comment by pckroon** [comment]
Remove it, since it slows down the FF
